src/generic_core/uproxy_core.ts

```diff
@@ -655,15 +655,64 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
       };
       const MAX_INSTALLS = 5;
       return retry(install, MAX_INSTALLS);
-    }).then((output: string) => {
+    }).then((cloudNetworkData :any) => {
```

Comment:
Just added a comment based on our discussion.  We'll do this later, it requires some more changes to uProxy-lib